app/Http/Controllers/Admin/RecipesController.php

```diff
@@ -4,8 +4,61 @@
 
 use App\Http\Controllers\Controller;
 use Illuminate\Http\Request;
+use Illuminate\Support\Facades\DB;
+use App\Models\Recipe;
+use App\Models\User;
+use App\Models\Category;
 
 class RecipesController extends Controller
 {
-    //
+    private $recipe;
+
+    public function __construct(Recipe $recipe){
+        $this->recipe = $recipe;
+    }
+
+    public function show(){
```

Comment:
Can you try below if it will output the same as what you expected?
```
public function show()
    {
        // Eager loading related data
        $all_recipes = Recipe::with(['category', 'country', 'eatingPreference', 'mealType', 'prefOccasion'])
            ->latest()
            ->withTrashed()
            ->paginate(10);

        return view('admin.list-of-recipes', compact('all_recipes'));
    }
```

Reply:
@MayukoMita does this mean that the team won't be making these models?

Reply:
@kredo-school At this moment we are not planning to... Is that going to be a problem?